src/openMVG/sfm/sfm_data_graph_utils.cpp

```diff
@@ -0,0 +1,139 @@
+// This file is part of OpenMVG, an Open Multiple View Geometry C++ library.
+
+// Copyright (c) 2015,2017 WhuAegeanSea, Pierre Moulon.
+
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "openMVG/sfm/sfm_data_graph_utils.hpp"
```

Comment:
Include must be in alphabetical ordering, except the first one that relate the cpp to the hpp.

You will have this ordering:
```
#include "openMVG/sfm/sfm_data_graph_utils.hpp"

#include "openMVG/features/feature.hpp" // Not sure that this header is required
#include "openMVG/graph/connectedComponent.hpp"
#include "openMVG/graph/graph.hpp"
#include "openMVG/graph/graph_builder.hpp"
#include "openMVG/matching/indMatch.hpp"
#include "openMVG/sfm/sfm_data.hpp"
#include "openMVG/sfm/pipelines/sfm_matches_provider.hpp"
#include "openMVG/sfm/sfm_filters.hpp"
#include "openMVG/stl/stl.hpp"
#include "openMVG/types.hpp"

```

Reply:
@pmoulon Thanks for you reply, I have fixed these issues.